File: BlackJack.WEB/ClientApp/src/app/shared/entities/game.views/start.game.view.ts
import { GameStateType } from '../../enums/game-state-type';

export class StartGameView {
  id?: string;
  wonName?: string;
  gameState?: GameStateType;
  countOfBots?: number;
  player?: PlayerStartGameView;

  constructor() {
    this.player = new PlayerStartGameView();
  }
}

export class PlayerStartGameView {
  userName?: string;
  playerId?: string;
  balance?: number;
  bet?: number;
}

File: BlackJack.ViewModels/GameViews/StartGameView.cs
using BlackJack.DataAccess.Entities;
using BlackJack.DataAccess.Enums;
using System;

namespace BlackJack.ViewModels.GameViews
{
    public class StartGameView
    {
        public Guid Id { get; set; }
        public string WonName { get; set; }
        public int CountOfBots { get; set; }
        public GameState GameState { get; set; }

        public PlayerStartGameView Player { get; set; }
    }

    public class PlayerStartGameView
    {
        public string PlayerId { get; set; }
        public string UserName { get; set; }
        public decimal Balance { get; set; }
        public decimal Bet { get; set; }
    }
}
